### PR TITLE
refactor: extract layout to new design and remove unused code

### PR DESCRIPTION
**Pull request by juanmanso**
## Description

There are a bunch of files changed and commits but the goal of this PR is:

- Remove dead code from scaffolding template
- Extract Headers and Footers to new layout
- Tweak metaheader, manifest and other related assets to new app
- Begin refactor and further tweaking of connect/login button
- Accomodate navlinks to fit into header
- Create large logo component to be shared across footer and header

For that we made also some changes along the way to:

- Button component
- `_app.tsx` page by turning it into `Layout` and `Providers` components

### CODE REVIEW

File: packages/nextjs/layout/Providers.tsx
Comment (fabriziogianni7) [inline packages/nextjs/layout/Providers.tsx:1]: I like this

File: packages/nextjs/pages/_app.tsx [hunk 1]
```diff
@@ -1,71 +1,30 @@
...
-            </NvmProvider>
-          </RainbowKitProvider>
-        </WagmiConfig>
-      </RobotsProvider>
-    </Web3AuthProvider>
+    <Layout>
+      {isLanding ? (
+        <Landing />
+      ) : (
+        <div className="flex flex-col min-h-screen">
```
Comment (fabriziogianni7) [inline packages/nextjs/pages/_app.tsx:17]: maybe that div can be a single component?

File: packages/nextjs/ui/Header/LoginButton.tsx
Comment (fabriziogianni7) [inline packages/nextjs/ui/Header/LoginButton.tsx:1]: question:
Is this login related with rainbowkit login?

**Review by fabriziogianni7 — APPROVED** [review]
LGTM
Just left some comments

File: packages/nextjs/pages/_app.tsx (hunk 1 above)
Comment (juanmanso) [reply packages/nextjs/pages/_app.tsx:17]: Yes it can, I will further refactor this in the future when I remove the bypass of the `router.pathname` check. Good call though

File: packages/nextjs/ui/Header/LoginButton.tsx
Comment (juanmanso) [reply packages/nextjs/ui/Header/LoginButton.tsx:1]: Yes it is. If you look at the connect component the baseline is the same:

- https://github.com/keyko-io/botblock-mvp/blob/ee0bc0ea9c25d0e55e776c40fc84a6833e8e8011/packages/nextjs/components/Header/components/RainbowKitCustomConnectButton.tsx#L30-L32
